controllers/shipwrightbuild_controller.go

```diff
@@ -69,7 +79,75 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	logger := r.Logger.WithValues("namespace", req.Namespace, "name", req.Name)
 	logger.Info("Starting resource reconciliation...")
 
-	// retrieving the ShipwrightBuild instance requested for reconciliation
+	// See if tekton is there
+	_, tektonAPIErr := r.CRDClient.CustomResourceDefinitions().Get(ctx, "taskruns.tekton.dev", metav1.GetOptions{})
+	tektonOpCRD, tektonOPErr := r.CRDClient.CustomResourceDefinitions().Get(ctx, "tektonconfigs.operator.tekton.dev", metav1.GetOptions{})
+	switch {
+	case tektonOPErr == nil && tektonAPIErr == nil:
+		logger.Info("Tekton is installed, and the Tekton Operator is also installed.")
```

Comment:
Since Tekton is growing, we should be more specific:
```suggestion
		logger.Info("Tekton Pipelines is installed, and the Tekton Operator is also installed.")
```